tests/raftstore_cases/test_split_region.rs

```diff
@@ -267,10 +267,10 @@ fn test_auto_split_region<T: Simulator>(cluster: &mut Cluster<T>) {
             Ok(true)
         })
         .expect("");
-    assert!(size <= REGION_SPLIT_SIZE);
-    // although size may be smaller than REGION_SPLIT_SIZE, but the diff should
+    assert!(size >= REGION_SPLIT_SIZE);
```

Comment:
for current implementation, the region size is a little bit larger than REGION_SPLIT_SIZE, and before is a little bit smaller than REGION_SPLIT_SIZE. Is that alright? @zhangjinpeng1987 @overvenus

Reply:
To keep consistency, in `on_kv` function, you should accumulate `current_size` first.